content/projects/jcasc/json-schema.adoc

```diff
@@ -0,0 +1,46 @@
+---
+layout: project
+title: "Jenkins Configuration as Code"
+section: projects
+tags:
+- jenkins-configuration-as-code
+- jcasc
+opengraph:
+  image: /images/logos/JCasC/JCasC.png
+links:
+  gitter: 'jenkinsci/configuration-as-code-plugin'
+  github: 'jenkinsci/configuration-as-code-plugin'
+  meetings: "/projects/jcasc/#office-hours-meeting"
+
+---
+
+THe JSON Schema is structured to validate the yaml files that are loaded via JCasC.The structure and validation of the Schema is done based on the user-installed plugins.
+
+=== Version
+
+The Schema uses JSON draft v07. 
+http://json-schema.org/draft-06/schema#
+
+
+=== Progress
+
+* The JSON Schema is currently not working.
+* We are working on it and will release a beta version.
+
+=== Office Hours meeting
```

Comment:
I don't see how the office hours is relevant to the JSON schema documentation.  I recommend removing that portion, since I assume it duplicates material elsewhere in the site.